src/Microsoft.DncEng.SecretManager/SecretTypes/AzureDevOpsAccessToken.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DncEng.CommandLineLib;
+
+namespace Microsoft.DncEng.SecretManager.SecretTypes
+{
+    [Name("azure-devops-access-token")]
+    public class AzureDevOpsAccessToken : GitHubAccountInteractiveSecretType<AzureDevOpsAccessToken.Parameters>
+    {
+        private readonly TimeSpan _rotateBeforeExpiration = new TimeSpan(-15, 0, 0, 0);
```

Comment:
I would prefer TimeSpan.FromDays here. I... think this is negative 15 days, but its not super clear without looking up the constructor parameters.

Reply:
you are right, it looks better with TimeSpan.FromDays, it's updated